openrtb/macrosubs_bench_test.go

```diff
@@ -106,3 +107,7 @@ func fakeBidResponse() *openrtb.BidResponse {
 	}
 	return bidRes
 }
+
+func fakeAuctionResult() *auctionResult {
```

Comment:
This function is not necessary, the type is simple enough to just use directly. Consider my other comment about make the type a float to be even simpler.